### PR TITLE
Add tests for snapshot and clone using KMIP (Thales)

### PR DESCRIPTION
**Pull request by rachael-george**
Add tests for PVC snapshot and restore and PVC cloning for encrypted PVCs using KMIP (Thales CipherTrust Manager)

Signed-off-by: rachael-george <rgeorge@redhat.com>

### CODE REVIEW

File: tests/manage/pv_services/pv_encryption/test_encrypted_rbd_pvc_snapshot.py [hunk 1]
```diff
@@ -56,6 +56,15 @@
                 "v2", kmsprovider, False, marks=pytest.mark.polarion_id("OCS-2613")
             ),
         ]
+    if Version.coerce(config.ENV_DATA["ocs_version"]) >= Version.coerce("4.12"):
```
Comment (petr-balogh) [inline tests/manage/pv_services/pv_encryption/test_encrypted_rbd_pvc_snapshot.py:59]: Please use this approach of comparing versions:
https://github.com/red-hat-storage/ocs-ci/blob/de5f02841523d4defa950e7bc09735333e6a09b4/ocs_ci/deployment/baremetal.py#L203

**Comment by nehaberry** [comment]
Hi @rachael-george 

What is preventing from merging this one? Just the approvals or more changes are required?

File: tests/manage/pv_services/pv_encryption/test_encrypted_rbd_pvc_snapshot.py (hunk 1 above)
Comment (rachael-george) [reply tests/manage/pv_services/pv_encryption/test_encrypted_rbd_pvc_snapshot.py]: Done

**Comment by rachael-george** [comment]
> Hi @rachael-george
> 
> What is preventing from merging this one? Just the approvals or more changes are required?

There was a small change for checking the version, I have made the changes, but would need help with testing the PR

**Pull request closed without merging**